src/client/components/localisation/skybox/sky_shader.ts

```diff
@@ -0,0 +1,207 @@
+import { observable } from 'mobx'
+import { Vector3 } from 'three'
+
+interface SkyUniforms {
+  luminance: {value: any}
+  turbidity: {value: any}
+  rayleigh: {value: any}
+  mieCoefficient: {value: any}
+  mieDirectionalG: {value: any}
+  sunPosition: {value: any}
+}
+
+export class SkyShader {
+
+  @observable public uniforms: SkyUniforms
+  @observable public vertexShader: string
+  @observable public fragmentShader: string
+
+  constructor() {
+    this.uniforms = {
+      luminance: { value: 1 },
+      turbidity: { value: 2 },
+      rayleigh: { value: 1 },
+      mieCoefficient: { value: 0.005 },
+      mieDirectionalG: { value: 0.8 },
+      sunPosition: { value: new Vector3() },
+    }
+
+    this.vertexShader = [
```

Comment:
Ok, I will change it, I thought it would be ok seeing as the files entire job was to just load it, and pretty much no maintenance would be needed.